### PR TITLE
Adding clarification for scripts that fail when PAT's aren't authorized

### PR DESCRIPTION
**Pull request by cknowles-admin**
Not all scripts fail on bad pat - and several fail with a proper error message.  Adding exception handling to those that failed in non-obvious ways.  

org_audit_licensefile.py
org_dependency_search.py
org_repo_perms.py

This is usually a thing with the scripts that are poking the graphql endpoints.

### CODE REVIEW

File: org_dependency_search.py [hunk 1]
```diff
@@ -136,6 +136,8 @@ def run_query(org, repo, headers, url):
             raise Exception(
                 f"Query failed to run by returning code of" f" {request.status_code}. {query}"
             )
+        if "errors" in jsonified.keys():
```
Comment (ctbmozilla-admin) [inline org_dependency_search.py:139]: Do all errors return with a `errors` key even if there's only a single error?

**Review by ctbmozilla-admin — APPROVED** [review]
Aside from my question, my suggestion would be to add some sort of "Error: " prefix before outputting the raw error messages, but that's mostly window dressing and up to you. Otherwise looks good to me! +1

File: org_dependency_search.py (hunk 1 above)
Comment (cknowles-admin) [reply org_dependency_search.py:139]: So, the errors field in the json is an array - I am only reporting the FIRST error - but "errors" is correct.

Comment (cknowles-admin) [reply org_dependency_search.py:139]: And modified things to be better about declaring errors - probably looks better - value your opinion.

Comment (ctbmozilla-admin) [reply org_dependency_search.py:139]: Looks good, merge dat!

**Pull request merged**